src/java.security.jgss/share/classes/sun/security/jgss/wrapper/GSSCredElement.java

```diff
@@ -42,9 +42,14 @@
     long pCred; // Pointer to the gss_cred_id_t structure
     private GSSNameElement name = null;
     private GSSLibStub cStub;
+    public boolean isDefCred;
```

Comment:
Are you sure you want this to be public? Especially since it's not final! What if someone else changes it?